ideascube/tests/test_widgets.py

```diff
@@ -54,3 +54,41 @@ def test_filtered_lang_select_with_pre_selected(user):
         ])
     rendered = widget.render_options(['fr'])
     assert rendered == expected
+
+
+def test_richtext_entry():
+    rte = RichTextEntry()
+
+    rendered_rte = rte.render('richtextentry', None,
+                              attrs={'id':'id_richtextentry'})
+    assert rendered_rte == '\n'.join([
+        '<div class="tinymce_editable" id="richtextentry"></div>',
+        ('<input id="id_richtextentry" name="richtextentry" type="hidden"'
+         ' value="" />'),
+        '<script type="text/javascript">tinymce.init({',
+        'selector: "#richtextentry",',
+        'inline: true,',
+        'menubar: false,',
+        'toolbar: false,',
+        'plugins: "autolink",',
+        'hidden_input: false',
+        '});</script>'])
+
+
+def test_richtext_entry_with_content():
+    rte = RichTextEntry()
+
+    rendered_rte = rte.render('richtextentry', "<p>A text</p>",
+                              attrs={'id':'id_richtextentry'})
```

Comment:
Not anymore.
I was using it in previous code version and the `id` has to be specify in the `attrs` dict.
But I'm not using it anymore, so it should be removed in the test.